### PR TITLE
Schema mapping itests

### PR DESCRIPTION
**Pull request by bharadwaj-aditya**
1. Corrected readme
2. Added schema mapping integration tests

### CODE REVIEW

File: v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md [hunk 1]
```diff
@@ -24,26 +24,23 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
...
-* **maxConnections** (Maximum number of connections to Source database per worker): Configures the JDBC connection pool on each worker with maximum number of connections. Use a negative number for no limit. Default value is 100.
-* **disabledAlgorithms** (Disabled algorithms to override jdk.tls.disabledAlgorithms): Comma-separated algorithms to disable. If this value is set to `none` then no algorithm is disabled. Use with care, because the algorithms that are disabled by default are known to have either vulnerabilities or performance issues. (Example: SSLv3, RC4).
-* **extraFilesToStage** (Extra files to stage in the workers): Comma separated Cloud Storage paths or Secret Manager secrets for files to stage in the worker. These files will be saved under the `/extra_files` directory in each worker (Example: gs://your-bucket/file.txt,projects/project-id/secrets/secret-id/versions/version-id).
+* **tables** (Comma seperated list of tables to migrate): Tables that will be migrated to Spanner. Leave this empty if all tabels are to be migrated. (Example: table1,table2).
```
Comment (Deep1998) [inline v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md:39]: tables*

File: v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    SpannerAsserts.assertThatStructs(companySpanner)
+        .hasRecordsUnorderedCaseInsensitiveColumns(companyMySQL);
+    SpannerAsserts.assertThatStructs(companySpanner).hasRows(companyMySQL.size());
+
+    List<Map<String, Object>> employeeMySQL =
+        mySQLResourceManager.runSQLQuery(
+            "SELECT employee_id, company_id, employee_name, employee_address as employee_address_sp FROM employee");
+    ImmutableList<Struct> employeeSpanner =
+        spannerResourceManager.readTableRecords(
+            "employee", "employee_id", "company_id", "employee_name", "employee_address_sp");
```
Comment (Deep1998) [inline v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java:107]: Shouldnt the table name be employee_sp?

File: v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTransformationIT.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    SpannerAsserts.assertThatStructs(companySpanner)
+        .hasRecordsUnorderedCaseInsensitiveColumns(companyMySQL);
+    SpannerAsserts.assertThatStructs(companySpanner).hasRows(companyMySQL.size());
+
+    List<Map<String, Object>> employeeMySQL =
+        mySQLResourceManager.runSQLQuery(
+            "SELECT employee_id, company_id, employee_name, employee_address as employee_address_sp FROM employee");
+    ImmutableList<Struct> employeeSpanner =
+        spannerResourceManager.readTableRecords(
+            "employee", "employee_id", "company_id", "employee_name", "employee_address_sp");
```
Comment (Deep1998) [inline v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTransformationIT.java:107]: Same here: Shouldnt the table name be employee_sp?

File: v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java
Comment (Deep1998) [inline v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java:1]: This file seems to be identical to SessionSchemaMappedWithTransformationsIT.

File: v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java (hunk 2 above)
Comment (bharadwaj-aditya) [reply v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java:107]: fixed.

File: v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md (hunk 1 above)
Comment (bharadwaj-aditya) [reply v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md:39]: fixed.

File: v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java
Comment (bharadwaj-aditya) [reply v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SessionSchemaMapperWithTableFilterIT.java:1]: corrected.

**Review by Deep1998 — APPROVED** [review]
LGTM

**Pull request merged**